lm_eval/tasks/unitxt/tasks.classification.multi_class.yaml

```diff
@@ -0,0 +1,26 @@
+group:
+- unitxt
+task: TASK_PLACEHOLDER
+dataset_path: unitxt/data
+dataset_name: card=CARD_PLACEHOLDER,template=TEMPLATE_PLACEHOLDER
+output_type: generate_until
+training_split: train
+validation_split: test
+doc_to_text: '{{source}}'
+doc_to_target: target
+process_results: !function 'unitxt_wrapper.process_results'
+generation_kwargs:
+  until:
+  - </s>
+metric_list:
+- metric: unitxt_f1_micro
+  aggregation: unitxt
+  higher_is_better: true
+- metric: unitxt_accuracy
+  aggregation: unitxt
+  higher_is_better: true
+- metric: unitxt_f1_macro
+  aggregation: unitxt
+  higher_is_better: true
+metadata:
+  version: 1.0
```

Comment:
Should be rename to something like 
`_tasks_classification_multi_class_yaml`. Since it's a template, not naming it with `.yaml` file extension will avoid it from being read as a task.

Reply:
Sure. I will rename all the files.